src/MultiplicationTable.ts

```diff
@@ -1,5 +1,48 @@
 export class MultiplicationTable {
   public render(start: number, end: number): string{
-    return '1*1=1'
+    const isValid = this.isValidInput(start, end)
+
+    if (isValid) {
+      return this.GenerateMulTable(start, end)
+    }else{
+      return ""
+    }
+  }
+
+  private isValidInput(start: number, end: number): boolean{
+    return this.IsStartLessEqualToEnd(start, end) && this.IsInRange10(start, end)
+  }
+
+  private IsStartLessEqualToEnd(start: number, end: number): boolean{
+    return start <= end
+  }
+
+  private IsInRange10(start: number, end: number): boolean{
+    return start >= 1 && end <= 10
+  }
+
+  // 2, 4 => [2,2], [2,3], [2,4]
+  private GenerateMulTable(start: number, end: number): string{
```

Comment:
'GenerateMultiplicationTable' is clearer than 'GenerateMulTable' as a method name.